python/desc/imsim/instcat_tools.py

```diff
@@ -0,0 +1,176 @@
+"""
```

Comment:
Does this code replace imsim_truth.py in master?  Also does it close #85?

Reply:
yes, I suppose it does on both counts.  I'll remove the imsim_truth.py code.